src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calc = {
+    result: 0,
+
+    add(objResult, num) {
+      return objResult + num;
+    },
+
+    subtract(objResult, num) {
+      return objResult - num;
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=` to subtract something from an existing value. Instead of returning a new value, you should update `this.result` directly. Example: `this.result -= num;`